### ISSUE TITLE
Source-Build: Eliminate Microsoft.DotNet.Cli.CommandLine in CLI

### ISSUE THREAD

**Issue by nguerrera** [dotnet/cli#11441]
cc @dseefeld 

CLI Prebuilts |   |   |
-- | -- | --
**Package** | **Version** | **Status**  
Microsoft.DotNet.Cli.CommandLine | 0.1.1 | Use Source Built Version

**Comment by nguerrera** [comment]
@dseefeld I can't reproduce this one:

Microsoft.DotNet.Cli.CommandLine | 0.1.1 | Use Source Built Version
-- | -- | --

Everything seems to be wired up correctly.

```
nicholg@nicholg-macbook:~/src/cli/.nuget/packages
$ ls microsoft.dotnet.cli.commandline/
1.0.0-preview.19208.1/

nicholg@nicholg-macbook:~/src/cli
$ ack MicrosoftDotNetCliCommandLine
eng/Versions.props
33:    <MicrosoftDotNetCliCommandLinePackageVersion>1.0.0-preview.19208.1</MicrosoftDotNetCliCommandLinePackageVersion>

src/dotnet/dotnet.csproj
67:    <PackageReference Include="Microsoft.DotNet.Cli.CommandLine" Version="$(MicrosoftDotNetCliCommandLinePackageVersion)" />

test/dotnet.Tests/dotnet.Tests.csproj
82:    <PackageReference Include="Microsoft.DotNet.Cli.CommandLine" Version="$(MicrosoftDotNetCliCommandLinePackageVersion)" />
```
```

**Comment by nguerrera** [comment]
https://github.com/dotnet/toolset/pull/1088 is out to remove System.Data.SqlClient from source build.

**Comment by nguerrera** [comment]
https://github.com/dotnet/core-sdk/pull/2232 is out to remove the DotnetHostResolver packages from core-sdk

**Comment by nguerrera** [comment]
cli-migrate is being removed from the product. cc @livarcocc

**Comment by nguerrera** [comment]
So status is:

* **cli**: Looks like false positive. @dseefeld can you look at https://github.com/dotnet/cli/issues/11441#issuecomment-497101652

* **cli-migrate**: Doesn't matter since it will be removed from product

* **toolset* in PR: dotnet/toolset#1088 i

* **core-sdk** in PR: dotnet/core-sdk#2232 i

**Comment by nguerrera** [comment]
Moving to individual issues per instructions. This one now tracks only the cli issue that I think is a false positive.

**Comment by nguerrera** [comment]
The fixes to the other issues are merged. This is the last one, and I'm waiting to understand as noted above.

**Comment by dseefeld** [comment]
@nguerrera I investigated this one.  In source-build, we have a patch that is pinning the version to 0.1.1.  I added a PR to remove this patch: https://github.com/dotnet/source-build/pull/1058

**Comment by nguerrera** [comment]
Aha! Thanks!

**Comment by nguerrera** [comment]
Fixed by dotnet/source-build#1058
